bin/webpack.js

```diff
@@ -1,36 +1,45 @@
 #!/usr/bin/env node
+
 /*
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-"use strict";
-
 var path = require("path");
 var resolveCwd = require("resolve-cwd");
-
+// Local version replace global one
 var localCLI = resolveCwd.silent("webpack-cli/bin/webpack");
 
+if (process.argv.slice(2).includes("init")) {
```

Comment:
`.includes` is not supported by Node 5 and 4. We need to change it. Also below
http://node.green/#ES2015-built-ins-well-known-symbols-Symbol-match--String-prototype-includes